internal/collectors/lscpu.go

```diff
@@ -0,0 +1,36 @@
+package collectors
```

Comment:
Is this whole thing used in other places besides the CPU collector? Maybe it would make sense to have all of this inside of the CPU collector file.

Reply:
In the current hwinfo implementation, lscpu is used to fetch hypervisor and clusters data as well. Since we will be working on improving how we collect these metrics in another PR and also exploring the json output for lscpu, I think we should revisit this then and see if it still makes sense to have it as a separate file